lxd/db/generate/db/stmt.go

```diff
@@ -135,7 +135,12 @@ func (s *Stmt) objects(buf *file.Buffer) error {
 	for _, field := range fields {
 		if field.Config.Get("order") != "" {
 			if field.IsScalar() {
-				orderBy = append(orderBy, lex.Plural(lex.Snake(field.Name))+".id")
+				table, _, err := field.ScalarTableColumn()
```

Comment:
Is this an unrelated bug fix? If so can it be in a separate PR please?

Reply:
To be honest, this should be taken care of by the next PR I have coming up so I've just removed it outright for now.